azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/error/ErrorCode.kt

```diff
@@ -14,6 +14,8 @@ internal class ErrorCode : ExpandableStringEnum<ErrorCode?>() {
         val TURN_CAMERA_OFF_FAILED = fromString("turnCameraOffFailed")
         val TURN_MIC_ON_FAILED = fromString("turnMicOnFailed")
         val TURN_MIC_OFF_FAILED = fromString("turnMicOffFailed")
+        val UNKNOWN_ERROR = fromString("unknownError")
```

Comment:
This error code has been removed. Please delete this.

Reply:
Ok